addon/mixin.js

```diff
@@ -10,6 +11,7 @@ const bound = function(fnName) {
     if (fn) { // https://github.com/zeppelin/ember-click-outside/issues/1
       return fn.bind(this);
     }
+    return
```

Comment:
Is this needed here?

Reply:
It was added to address the new rule `require-return-from-computed` in `eslint-plugin-embed`: https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/require-return-from-computed.md

We could alternatively disable the rule (inline, or in the lint config).

Reply:
You know what? It's fine now. I'll revisit the lint config later.